src/pybert/pybert.py

```diff
@@ -53,43 +44,38 @@
     Property,
     Range,
     String,
-    Trait,
+    cached_property,
 )
+from traits.etsconfig.api import ETSConfig
```

Comment:
This needs to remain at the top of the import stack, because it's important that the (currently commented out) lines that use it execute before anything else.